app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DisposalDetails.scala

```diff
@@ -89,13 +89,13 @@ object DisposalDetails {
     )
   }
 
-  private def getInitialGainOrLoss(calculatedTaxDue: CalculatedTaxDue): (BigDecimal, BigDecimal) = {
-    val value = calculatedTaxDue.initialGainOrLoss
+  private def getInitialGainOrLoss(calculatedTaxDue: CalculatedTaxDue): (Option[BigDecimal], Option[BigDecimal]) = {
```

Comment:
this should now not take in a `CalulatedTaxDue`, but the `initialGainOrLoss` `AmountInPence` from a `CompleteReturn`. And then we should only populate the values in DES is that field is defined. This'll probably mean you now need to stick an optional `initialGainOrLoss`field into the `CompleteReturn`

Reply:
changed to use the CompleteReturn to find existing vals from